tools/storm/utils/trident.go

```diff
@@ -135,17 +138,20 @@ func CheckTridentService(client *ssh.Client, env TridentEnvironment, timeout tim
 			return nil, nil
 		},
 	)
+	if reconnectNeeded != nil && *reconnectNeeded {
+		return fmt.Errorf("SSH connection needs to be re-established")
+	}
 	if err != nil {
 		return fmt.Errorf("trident service is not in expected state: %w", err)
 	}
 
 	return nil
 }
 
-func checkTridentServiceInner(client *ssh.Client, serviceName string) error {
+func checkTridentServiceInner(client *ssh.Client, serviceName string, expectSuccessfulCommit bool) (bool, error) {
```

Comment:
could you add a doc comment about the returns?

Reply:
done